src/cascadia/QueryExtension/ILLMProvider.idl

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.Terminal.Query.Extension
+{
+    interface ILLMProvider
+    {
+        void ClearMessageHistory();
```

Comment:
Wondering if an `ExportMessageHistory` method would be useful to add onto this interface? Customers may want to have a convenient way to have a record of a conversation.

Reply:
Ooh that's a good one! I don't think the LLMProvider should be the one responsible for implementing that though - I think the ExtensionPalette can handle that (it also has a storage of the message history since it needs it to display in the UI)

Reply:
Gah, my bad- yes, this should be an operation on the palette (not the provider!).